container_files/tutorials/Virtual manipulation of datasets.ipynb

```diff
@@ -0,0 +1,924 @@
+       "symmetric                     NaN                  NaN                  NaN   \n",
+       "marked                          1                  NaN                  NaN   \n",
+       "such                          NaN                    1                    1   \n",
+       "hopfield                      NaN                  NaN                  NaN   \n",
+       "only                            1                  NaN                  NaN   \n",
+       "\n",
+       "           Hopfield network  \n",
+       "_rowName                     \n",
+       "maximum                 NaN  \n",
+       "nervous                 NaN  \n",
+       "random                  NaN  \n",
+       "studied                 NaN  \n",
+       "neurons                 NaN  \n",
+       "symmetric               NaN  \n",
+       "marked                  NaN  \n",
+       "such                    NaN  \n",
+       "hopfield                  4  \n",
+       "only                    NaN  "
+      ]
+     },
+     "execution_count": 18,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "mldb.query(\"SELECT * FROM sampled_tokens\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": false,
+    "scrolled": true
+   },
+   "source": [
+    "The above code would be simlar to the following SQL function:\n",
+    "```python\n",
+    "mldb.query(\n",
+    "    \"\"\"\n",
+    "    SELECT * \n",
+    "    FROM sample(\n",
+    "        transposed_concepts, \n",
+    "        {\n",
+    "            rows: 10,\n",
+    "            withReplacement: False\n",
+    "        }\n",
+    "    )                      \n",
+    "    \"\"\"\n",
+    ") \n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Again, the two methods provide the same desired outcome, allowing you to choose how to best manipulate your data."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Using datasets vs. SQL functions\n",
+    "As seen above, the two different ways to either transpose or sample data are equivalent. It is recommended to use Dataset types instead of SQL functions when the created table will be reused or called later in the program.\n",
+    "\n",
+    "As seen in this tutorial, MLDB allows you to virtually manipluate data in multiple ways. This grants greater flexibility when constructing models and analyzing complex data."
+   ]
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Python 2",
+   "language": "python",
+   "name": "python2"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 2
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython2",
+   "version": "2.7.6"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 0
+}
```

Comment:
Manque un bout: Where to next? Check out the other Tutorials and Demos.

comme dans les autres tutoriels